doc/modules/classes.rst

```diff
@@ -1387,6 +1387,30 @@ Low-level methods
    utils.validation.column_or_1d
    utils.validation.has_fit_parameter
 
+
+.. _experimental_ref:
+
+:mod:`sklearn.experimental`: Experimental functionality
+=======================================================
+
+.. automodule:: sklearn.experimental
```

Comment:
This is not changed because we haven't agreed on where to put it?

Reply:
> This is not changed because we haven't agreed on where to put it?

yes